.gitignore

```diff
@@ -1 +1,12 @@
 node_modules
+
+# Stuff to ignore related to local testing with Jekyll
+_site
+.sass-cache
+.jekyll-cache
+.jekyll-metadata
+vendor
+
+# Stuff to ignore related to emacs
```

Comment:
I don't mind having this in the repo, but generally I would put this sort of IDE specific stuff in a global gitignore local to the machine.